### DISCUSSION TITLE
如何撰写一个 ASF 孵化器提案？ | 夜天之书

### DISCUSSION THREAD

**Discussion by giscus** [tisonkun/blog#102] [Comments]
# 如何撰写一个 ASF 孵化器提案？ | 夜天之书

今天下午，我跟姜宁老师有一个线上直播会讨论与标题相同的问题，欢迎关注。   随着 Apache 软件基金会（ASF）在国内的深入发展，越来越多的项目希望藉由进入 ASF 孵化器孵化，来建设开源社群。在进入孵化器之前，项目发起人或其核心团队必须撰写一份孵化器提案来介绍项目的基本情况，以供孵化器项目管理委员会（Incubator Project Management Committee, IPMC）评

https://www.tisonkun.org/2023/12/23/asf-incubator-proposal-template/

**Comment by tanruixiang** [comment]
> 这个问题的意思是，项目是否只是“对 Apache 品牌的过度迷恋”而捐赠，而不是孵化器关注的按照 The Apache Way 建设社群。换句话说，是不是只想借 Apache 的品牌做营销。

写的时候品了好久什么是`A Excessive Fascination with the Apache Brand`，现在看到这个 `换句话说`才完全明白，很关键🤣

**Reply by tisonkun** [reply]
所以我才说“这个问题是整个提案模板里最容易被误会的一条”，因为我自己第一次直面这个问题的时候也有点晕 ..
